About.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="./assets/styles/about.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+<body>
+    <div class="main-container">
+    <head>
+        <div class="mobile-nav">
+            <div class="moblile-menu-icon">
+                <!-- <a id="mobile-icon-btn" href="#"> <img src="./assets/images/mobile_menu.png" alt="menu hamburger icon"></a> -->
+            </div>
+            <div id="myNav" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <div class="overlay-content">
+                    <a href="#portfoliostart" onclick="closeNav()">About</a>
+                    <a href="#about-social" onclick="closeNav()">Program</a>
+                    <a href="#contact" onclick="closeNav()">Join</a>
+                    <a href="#News" onclick="closeNav()">News</a>
+                    <a href="#contact" onclick="closeNav()">CC Campaign</a>
+                </div>
+            </div>
+            <div class="desk-nav">
+                <div class="logo-png">
+                    <a href="./home.html"> <img src="./assets/images/logopng.png" alt="logo image"> </a>
+                </div>
+                <div class="desk-nav-items">
+                    <ul>
+                        <li class="menu-item"><a href="./About.html"> About </a> </li>
+                        <li class="menu-item">Progam</li>
+                        <li class="menu-item">Join</li>
+                        <li class="menu-item">Sponsor</li>
+                        <li class="menu-item">News</li>
+                    </ul>
+                </div>
+            </div>
+            <nav class="flex-nav"><button onclick="openNav()">☰</button></nav>
+        </div>
+    </head>
+    <!-- coding the hero section here -->
+    <section class="hero-section">
```

Comment:
- [ ] Kindly make sure the content of the hero section for your about page mobile view matches the design guidelines given in terms of margins( i.e The section has too many margins), fonts (i.e Your fonts are too small,) and the font-weight so that your website can look better 😄 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190512599-80a3b1a0-9cc2-4f44-b277-28abca7ff87e.png)  | ![image](https://user-images.githubusercontent.com/63473584/190512883-e950ed35-9c63-4267-a62e-6c6c32dedbfe.png) |